Content.Shared/_Goobstation/MartialArts/Components/GrabStagesOverrideComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Content.Shared.Movement.Pulling.Systems;
+
+namespace Content.Shared._Goobstation.MartialArts.Components;
+
+/// <summary>
+/// Base component for martial arts that override the normal grab stages.
+/// Allows martial arts to start at more advanced grab stages like Hard grabs.
+/// </summary>
+public abstract partial class GrabStagesOverrideComponent : Component
+{
+    public GrabStage StartingStage = GrabStage.Hard;
```

Comment:
Might be worth makign this a datafield, to be more flexible.
```suggestion
    [DataField]
    public GrabStage StartingStage = GrabStage.Hard;
```
Won't require a change in your yaml if you keep the current default. Do check if you can just turn enums into datafields though--if you can't, ignore this suggestion.